File: Code/java/pattern7.java
import java.util.Scanner;

class Pattern7
{
    public static void main(String[] args)
    {
        int limit, i;
        Scanner scanner = new Scanner(System.in);

        System.out.println("Enter Limit: ");
        limit = scanner.nextInt();

        for (i = 0; i < limit; i++)
            System.out.println("  ".repeat(limit - i - 1) + " *".repeat(2 * i + 1));

        scanner.close();
    }
}

File: Code/cpp/pattern7.cpp
#include<iostream>
using namespace std;
int main()
{
    int limit, i, j;

    cout << "Enter Limit: ";
    cin >> limit;

    for (i = 0; i < limit; i++)
    {
        for (j = 0; j <  limit - i - 1; j++)
            cout << "  ";
        for (j = 0; j < 2 * i + 1; j++)
            cout << " *";
        cout << "\n";
    }
    return 0;
}